BUILD.md

```diff
@@ -100,9 +100,12 @@ The steps below will set up a Python virtual environment to run TCF.
    Refer to the [PREREQUISITES document](PREREQUISITES.md)
    for more details on the above variables
 
-5. Build the Python virtual environment and install TCF components into it:
+5. Create Python virtual environment, Build and Install install TCF
```

Comment:
Type . Remove duplicate install "Install install"

Reply:
Did you get a chance to try the standalone build with the changes in this PR

Reply:
Will correct the typo.
I have tried to install virtualenv on standalone build and tried pip install/uninstall of tcf python packages and it works fine.
The whole set of instructions for standalone build is not tried out.